samples/client/petstore/go-experimental/go-petstore/model_enum_class.go

```diff
@@ -13,6 +13,10 @@ import (
 	"encoding/json"
 )
 
+import (
+    "fmt"
+)
+
```

Comment:
The change looks good to me. I'll try to consolidate the multiple `import` later with another PR.